hbase-shell/src/main/ruby/shell/commands/alter.rb

```diff
@@ -72,6 +72,13 @@ def help
   hbase> alter 't1', CONFIGURATION => {'hbase.hregion.scan.loadColumnFamiliesOnDemand' => 'true'}
   hbase> alter 't1', {NAME => 'f2', CONFIGURATION => {'hbase.hstore.blockingStoreFiles' => '10'}}
 
+You can also set configuration setting with 'no_reopen_regions' to avoid regions RIT, which let the
+modification take effect after regions was reopened (Be careful, the regions of the table may be
+configured inconsistently If regions are not reopened after the modification)
+
+  hbase> alter 't1', METHOD => 'no_reopen_regions', CONFIGURATION => {'hbase.hregion.scan
```

Comment:
Minor : no_reopen_regions sounds a bit differently worded. Can we use something like 'avoid_reopening_regions' or 'dont_reopen_regions' ? or anything better ? I know dont without any apostrophe is still not good.

Reply:
We can also wait to see, if others are good with the current switch name.

Reply:
I believe 'avoid_reopening_regions' is better. It clearly conveys our purpose.